uni/lib/controller/local_storage/app_shared_preferences.dart

```diff
@@ -152,6 +154,19 @@ class AppSharedPreferences {
         .toList();
   }
 
+  static saveHiddenExams(List<Exam> newHiddenExams) async {
+    final prefs = await SharedPreferences.getInstance();
+    prefs.setStringList(
+        hiddenExams, newHiddenExams.map((exam) => json.encode(exam)).toList());
+  }
+
+  static Future<List<Exam>> getHiddenExams() async {
+    final prefs = await SharedPreferences.getInstance();
+    final List<String>? storedHiddenExam = prefs.getStringList(hiddenExams);
+    if (storedHiddenExam == null) return List<Exam>.empty();
```

Comment:
`List<Exam>.empty()` creates a fixed size list. This will generate an error `Unsupported operation: Cannot add to a fixed-length list` in the future when we try to add or remove exams from the list.

```suggestion
    if (storedHiddenExam == null) return <Exam>[];
```
A way to create an empty non-fixed size list